src/style.css

```diff
@@ -1 +1,51 @@
-/* add styles here */
+body {
+  margin: 0;
+}
```

Comment:
Avoid using tag names for styling (except `html` and `body`). It's a better practice to use classes for styling.